TSOClient/FSO.Server.Api.Core/Controllers/NhoodInfoController.cs

```diff
@@ -17,133 +17,117 @@ namespace FSO.Server.Api.Core.Controllers
     public class NhoodInfoController : ControllerBase
     {
         [HttpGet]
-        [Route("userapi/city/{shardid}/neighborhoods/all.json")]
-        public IActionResult GetAll(int shardid)
+        [Route("userapi/city/{shardId}/neighborhoods/all")]
+        public IActionResult GetAll(int shardId)
         {
             var api = Api.INSTANCE;
 
             using (var da = api.DAFactory.Get())
             {
-                var Nhoods = da.Neighborhoods.All(shardid);
-                if (Nhoods == null)
-                {
-                    var JSONError = new JSONNhoodError();
-                    JSONError.Error = "Neighborhoods not found";
-                    return ApiResponse.Json(HttpStatusCode.NotFound, JSONError);
-                }
+                var nhoods = da.Neighborhoods.All(shardId);
+                if (nhoods == null) return ApiResponse.Json(HttpStatusCode.NotFound, new JSONNhoodError("Neighborhoods not found"));
 
-                List<JSONNhood> NhoodJSON = new List<JSONNhood>();
-                foreach (var Nhood in Nhoods)
+                List<JSONNhood> nhoodJson = new List<JSONNhood>();
+                foreach (var nhood in nhoods)
                 {
-                    NhoodJSON.Add(new JSONNhood
+                    nhoodJson.Add(new JSONNhood
                     {
-                        Neighborhood_ID = Nhood.neighborhood_id,
-                        Name = Nhood.name,
-                        Description = Nhood.description,
-                        Color = Nhood.color,
-                        Town_Hall_ID = Nhood.town_hall_id,
-                        Icon_Url = Nhood.icon_url,
-                        Mayor_ID = Nhood.mayor_id,
-                        Mayor_Elected_Date = Nhood.mayor_elected_date,
-                        Election_Cycle_ID = Nhood.election_cycle_id
+                        neighborhood_id = nhood.neighborhood_id,
+                        name = nhood.name,
+                        description = nhood.description,
+                        color = nhood.color,
+                        town_hall_id = nhood.town_hall_id,
+                        icon_url = nhood.icon_url,
+                        mayor_id = nhood.mayor_id,
+                        mayor_elected_date = nhood.mayor_elected_date,
+                        election_cycle_id = nhood.election_cycle_id
                     });
 
                 }
-                var NhoodsJSON = new JSONNhoods();
-                NhoodsJSON.Neighborhoods = NhoodJSON;
-                return ApiResponse.Json(HttpStatusCode.OK, NhoodsJSON);
+                var nhoodsJson = new JSONNhoods();
+                nhoodsJson.neighborhoods = nhoodJson;
+                return ApiResponse.Json(HttpStatusCode.OK, nhoodsJson);
             }
         }
         [HttpGet]
-        [Route("userapi/neighborhoods/id/{Nhoodid}.json")]
-        public IActionResult GetByID(uint Nhoodid)
+        [Route("userapi/neighborhoods/{NhoodId}")]
```

Comment:
Case mismatch with parameter?

Reply:
Must have overlooked that typo, thanks for pointing it out 👍